web/src/main/java/org/springframework/security/web/webauthn/registration/WebAuthnRegistrationFilter.java

```diff
@@ -105,6 +105,24 @@ public WebAuthnRegistrationFilter(UserCredentialRepository userCredentials,
 		this.rpOptions = rpOptions;
 	}
 
+	/**
+	 * Sets the {@link RequestMatcher} used to trigger this filter.
```

Comment:
Please indicate the default `requestMatcher` value.

Please also add how this is different from the other `RequestMatcher` method. For example

```
Sets the ... to trigger this filter's the credential registration operation 
```